scheduler/src/cook/kubernetes/api.clj

```diff
@@ -675,31 +675,33 @@
 
 (defn launch-pod
   "Given a V1Pod, launch it."
```

Comment:
We should add a doctring explaining the behavior for pod submission failing and what the boolean return value means.

Reply:
Good call, done